crew

```diff
@@ -47,6 +47,22 @@ Usage:
 version #{CREW_VERSION}
 DOCOPT
 
+LICENSE = <<~LICENSESTRING
```

Comment:
Move to `const.rb`?

Reply:
Sure, if you'd like.

Reply:
Good suggestion @supechicken.  Also change to `puts CREW_LICENSE`.

Reply:
On second thought, it made more sense since the DOCOPT string is right above it

Reply:
Yeah, it doesn't have to be a crew constant.